### ISSUE TITLE
What's the difference between the new updated version and version published in paper?

### ISSUE THREAD

**Issue by kleinzcy** [microsoft/Semi-supervised-learning#28]
The resutls of two version is a little different especially on STL10(decreases a lot) and CIFAR100(improves a lot). I curious about the difference of two version. Maybe you can add the difference between two version into the readme.

Thanks.

**Comment by Hhhhhhao** [comment]
The main difference is that we adopt layer decay of learning rate, which we find very helpful for transferring pre-trained model in SSL. 

For STL10, you can also observe the results are lower than classic settings without using pre-trained models. We think better results on STL-10 could be achieved, but we need to search more hyper-parameters. If you are interested and found better settings, welcome to open a pull request.

**Comment by wyyfffff** [comment]
I found there are ImageNet in the datasets of USB docs: https://usb.readthedocs.io/en/main/benchmark.html#cv-datasets. but no config files in this repertory. Will ImageNet be added in later update?

**Comment by Hhhhhhao** [comment]
ImageNet is supported in USB but not officially included in the benchmark due to its long training time. Also its performance with our current training scheme is not very convincing. We are considering updating this paper (https://arxiv.org/abs/2208.05688) into USB for ImageNet, will be updated later.

**Comment by syorami** [comment]
@Hhhhhhao hi does this mean using the current codebase on imagenet could not get comparable performance with results in origin paper? I'm wondering whether to use this for experiments on imagenet.

**Comment by Hhhhhhao** [comment]
I think the ImageNet experiment will make no difference.

**Comment by syorami** [comment]
then what does "not convincing performance" mean?

**Comment by Hhhhhhao** [comment]
> then what does "not convincing performance" mean?

It means under USB's pre-training setting, it is expected to have higher accuracy as in https://arxiv.org/abs/2208.05688, but currently we do not have this performance.

**Comment by syorami** [comment]
Thanks for your explanation.
